src/Job/Message.php

```diff
@@ -136,9 +136,9 @@ public function __toString()
     }
 
     /**
-     * @return array
+     * @return mixed
      */
-    public function jsonSerialize()
+    public function jsonSerialize(): mixed
```

Comment:
You need to add the `\ReturnTypeWillChange` attribute instead to support PHP 7 and below.